### ISSUE TITLE
[$15][MSFT-178] [Challenge Feed] "View more challenges" button is not marked when user focus on it

### ISSUE THREAD

**Issue by veshu** [topcoder-platform/community-app#2850]
**Steps to Reproduce**
1. Go to https://www.topcoder.com/challenges and Login
2. Press Tab key until "View more challenges" button is selected

**Expected Result**
It should be marked with a border to easily identified

**Actual Result**
"View more challenges" button is not marked when user focus on it

**Device**: Desktop/Labtop
**Operating System**: Windows
**Browser**: Chrome

**WCAG Category**: 2. Operable
**WCAG Level**: AA
**WCAG Success Criterion**: 2.4.7 Focus Visible
**Screen Reader Used**: N/A
**Accessibility Audit Tool used**: 

**Attachment**
https://topcodermsft-my.sharepoint.com/personal/pd-topcoder_topcodermsft_onmicrosoft_com/Documents/Forms/All.aspx?cid=58da9b68%2Df2a3%2D45d2%2Dbbae%2D15abdfd0865b&FolderCTID=0x0120005C598A51049FC14CBA882E1AEE168F51&sortField=Modified&isAscending=true&id=%2Fpersonal%2Fpd%2Dtopcoder%5Ftopcodermsft%5Fonmicrosoft%5Fcom%2FDocuments%2FMSFT%2DTeams%2DQA%2FJune%202019%2F29%20Jun%2FTest%20Results%2FBug%20Videos%20%26%20Log%20Files%2Fcodejam%2DAccessibility%2F64

**Comment by ThomasKranitsas** [comment]
Contest https://www.topcoder.com/challenges/30096045 has been created for this ticket.<br/><br/>```This is an automated message for thomaskranitsas via Topcoder X```

**Comment by ThomasKranitsas** [comment]
Contest https://www.topcoder.com/challenges/30096045 has been updated - the new changes has been updated for this ticket.<br/><br/>```This is an automated message for thomaskranitsas via Topcoder X```

**Comment by romitgit** [comment]
Please assign this to me.

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30096045 has been updated - it has been assigned to Timor.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30097069 has been created for this ticket.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30097069 has been updated - it has been assigned to Timor.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by drasticdpk** [comment]
Operating System: Windows
Browser: Chrome
URL : https://test-community-app.topcoder-dev.com/challenges

**Comment by tosha5252** [comment]
Verified working

Env: Windows 10
Chrome and Edge

**Comment by crazyk07** [comment]
Payment task has been updated: https://software.topcoder.com/review/actions/ViewProjectDetails?pid=30097069<br/><br/>```This is an automated message for crazyk via Topcoder X```
